build-aux/m4/ax_boost_chrono.m4

```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
-#      http://www.gnu.org/software/autoconf-archive/ax_boost_chrono.html
+#     https://www.gnu.org/software/autoconf-archive/ax_boost_chrono.html
```

Comment:
A thought about dependencies updates where we're pulling in something from an upstream repository: is this something that could be done via a scripted diff? It's not clear to me how these changes were pulled in.

Reply:
The process for this (and all M4 dependencies) is:
- Click on the link in the M4 file's header.
- Click on the link on that page to load the "latest version".
- Copy the file contents and past it into my editor.

Reply:
Note that, for some of the macros, doing it as @str4d says guarantees that we have the right to redistribute, even though those files are technically not under a permissive license (see #2827).